src/puppetlabs/puppetdb/command.clj

```diff
@@ -407,8 +407,8 @@
 
 (defprotocol PuppetDBCommandDispatcher
   (enqueue-command
-    [this command version certname producer-ts payload]
-    [this command version certname producer-ts payload command-callback]
+    [this command version certname producer-ts payload compression]
+    [this command version certname producer-ts payload compression command-callback]
```

Comment:
we'll need to update pe-puppetdb-extensions for this as well, since sync uses this api